Tests/Clappr_Tests/Classes/Plugins/Core/MediaControl/MediaControlTests.swift

```diff
@@ -245,41 +266,6 @@ class MediaControlTests: QuickSpec {
                     }
                 }
 
-                /*
```

Comment:
Why we have this a commented test?

Reply:
hey @geisonfgf , I had just reply about this topic on nabil's comment. Can u take a look at it? = )